### ISSUE TITLE
[Request] Importing existing interactive video

### ISSUE THREAD

**Issue by philpav** [h5p/h5p-course-presentation#11]
Hello, 

it would be nice to be able to import an existing interactive video. Maybe by uploading the corresponding h5p file.

**Comment by icc** [comment]
I agree, that would be a really nice feature. Currently we're working on a copy&paste feature which I believe will help you do this. We haven't set any release date yet, but you should expect to see it in a release soon.

**Comment by otacke** [comment]
Closing, as the copy&paste feature that was the promised solution was released years ago.
